Okta.Xamarin/Okta.Xamarin/RenewResponse.cs

```diff
@@ -0,0 +1,52 @@
+﻿// <copyright file="RenewResponse.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using Newtonsoft.Json;
+using Okta.Xamarin.Models;
+
+namespace Okta.Xamarin
+{
+    /// <summary>
+    /// Represents data returned in response to a renew request.
+    /// </summary>
+    public class RenewResponse : Serializable
+    {
+        /// <summary>
+        /// Gets or sets the token type.
+        /// </summary>
+        [JsonProperty("token_type")]
+        public string TokenType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the expires in.
+        /// </summary>
+        [JsonProperty("expires_in")]
+        public int ExpiresIn { get; set; }
```

Comment:
Should we specify the time unit in the comments (i.e seconds)?

Reply:
We probably should; it wasn't clear to me what the actual unit was.  I'll add that.